src/app/containers/Timestamp/index.test.jsx

```diff
@@ -1,12 +1,10 @@
 import React from 'react';
```

Comment:
Should we not have a test here that has `isRelative` as `true`?

Reply:
Yes, they can't be snapshots, they'll need to be assertion-based tests, like in `ArticleTimestamp` https://github.com/bbc/simorgh/blob/latest/src/app/containers/ArticleTimestamp/index.test.jsx#L94  I've opened an issue here to add these: https://github.com/bbc/simorgh/issues/1666 This PR is a quick fix to prevent `latest` from being broken.